ndk/NDKProvider.tsx

```diff
@@ -41,8 +41,12 @@ const NDKProvider = ({
     const initNDK = async () => {
       const ndkInstance = new NDK({ explicitRelayUrls });
 
-      // Connect to the relays
-      await ndkInstance.connect();
+      try {
+        // Connect to the relays
+        await ndkInstance.connect();
+      } catch (error) {
+        console.error(error);
+      }
```

Comment:
This is to prevent the app from crashing if connecting to any of the relays fail.